services/openshift/scripts/add_insecure_registry

```diff
@@ -36,7 +36,7 @@ function init_insecure_reg()
         sed -i.orig "/# INSECURE_REGISTRY=*/c\INSECURE_REGISTRY=\"\"" ${docker_conf_file}
         # Get machine IP address
         local local_ip=$(get_ip_address)
-        sed -i.back "s/INSECURE_REGISTRY=\"\(.*\)\"/INSECURE_REGISTRY=\"\1 --insecure-registry 172.30.0.0\/16 --insecure-registry $local_ip\"/" ${docker_conf_file}
```

Comment:
That `/` is sed option. Also we have another PR #67 with update to this file.
# sed -i -e "s/<find_text>/<replace>/"